### PR TITLE
Add Mono to project list

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: _data/projects/mono-mono.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Mono
+desc: An open source implementation of Microsoft's .NET Framework based on the ECMA standards for C# and the Common Language Runtime.
+site: https://mono-project.com/
+tags:
+- C#
+- C
+- ".NET"
+upforgrabs:
+  name: good first issue
+  link: https://github.com/mono/mono/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/mono-mono.yml:10]: This label doesn't yet exist on their issue tracker

Comment (marek-safar) [reply _data/projects/mono-mono.yml:10]: What do you mean? The label exists but it's not yet used for any issue

Comment (shiftkey) [reply _data/projects/mono-mono.yml:10]: Oops, I guess I skimmed over it in the list - apologies!

**Comment by shiftkey** [comment]
I'll label this as `needs tasks` so please bump this when there are some tasks assigned and we can merge it in!

**Comment by marek-safar** [comment]
There are now few tasks available with the label

**Comment by ritwik12** [comment]
@marek-safar Great :)

**Comment by ritwik12** [comment]
@marek-safar Thanks :)

**Pull request merged**